protocols/kad/Cargo.toml

```diff
@@ -3,7 +3,7 @@ name = "libp2p-kad"
 edition = "2021"
 rust-version = "1.65.0"
 description = "Kademlia protocol for libp2p"
-version = "0.44.2"
+version = "0.44.3"
```

Comment:
You will also need to bump the version in the root manifest in our `workspace.dependencies`.